controller.go

```diff
@@ -372,6 +372,12 @@ func syncPV(pv *PV) {
 				}
 			} else {
 				// The PV must have been created with this ptr; leave it alone.
```

Comment:
That is also my interpretation of available.  Coarsely, it is available to be bound, but the set of PVCs that can bind is restricted to exactly one name.